### ISSUE TITLE
collation setting doesn't work

### ISSUE THREAD

**Issue by chris-han** [microsoft/azuredatastudio#17390]
Issue Type: <b>Bug</b>

created a project and change the 'ModelCollation' setting to 'Chinese_PRC_CI_AS' in project file .sqlprj, publish the project to azure.cn. the database Collation won't change.

Extension version: 0.12.0
Azure Data Studio version: azuredatastudio 1.32.0 (4a45ba7cf20dd4129f1a08e5e776dfb33e3d1d1e, 2021-08-16T18:08:28.086Z)
OS version: Windows_NT x64 10.0.22000

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-10875H CPU @ 2.30GHz (16 x 2304)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|63.87GB (41.25GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details>

**Comment by kisantia** [comment]
@chris-han did you set the ModelCollation to `<ModelCollation>Chinese_PRC_CI_AS</ModelCollation>` or `ModelCollation>2052,CI</ModelCollation>` in the .sqlproj file? Are you publishing to an existing database or is publish creating a new database?

There currently isn't a way in the ADS UI to change the project's collation, but when I changed my test project's collation in SSDT to Chinese_PRC_CI_AS in the properties dialog, the .sqlproj was updated with `ModelCollation>2052,CI</ModelCollation>`. When I published the project. the created database had the correct collation.
